### PR TITLE
Allow to torrent a single file

### PR DESCRIPTION
**Pull request by furstenheim**
#464

### CODE REVIEW

File: main/menu.js [hunk 1]
```diff
@@ -144,7 +156,12 @@ function showOpenTorrentAddress () {
 function getAppMenuTemplate () {
   var fileMenu = [
     {
-      label: 'Create New Torrent...',
+      label: 'Create New Torrent from File',
```
Comment (feross) [inline main/menu.js]: Should be 'Create New Torrent from File...'

File: main/menu.js [hunk 2]
```diff
@@ -144,7 +156,12 @@ function showOpenTorrentAddress () {
...
+      accelerator: 'CmdOrCtrl+F',
```
Comment (feross) [inline main/menu.js]: No shortcut needed for this one.

File: main/menu.js [hunk 3]
```diff
@@ -144,7 +156,12 @@ function showOpenTorrentAddress () {
 function getAppMenuTemplate () {
   var fileMenu = [
     {
-      label: 'Create New Torrent...',
```
Comment (feross) [inline main/menu.js]: This menu item should only be added on non OS X platforms.

File: main/menu.js [hunk 4]
```diff
@@ -144,7 +156,12 @@ function showOpenTorrentAddress () {
...
+      label: 'Create New Torrent from File',
+      accelerator: 'CmdOrCtrl+F',
+      click: showOpenSeedFile
+    },
+    {
+      label: 'Create New Torrent from Folder',
```
Comment (feross) [inline main/menu.js]: Need a conditional here. Should still be 'Create New Torrent...' on OS X.

Add '...' at the end for non OS X.

**Comment by feross** [comment]
Mostly looks good. See inline comments.

**Comment by furstenheim** [comment]
@feross I've addressed your comments in the last commit. I do not have OS X to check it there, but it should be alright.

**Comment by feross** [comment]
Merged -- nice work!

**Pull request merged**